packages/swapper/src/types.ts

```diff
@@ -48,6 +48,8 @@ export enum TradeQuoteError {
   SellAmountBelowMinimum = 'SellAmountBelowMinimum',
   // the fees exceed the sell amount
   SellAmountBelowTradeFee = 'SellAmountBelowTradeFee',
+  // the swapper has exceeded it's rate limit
```

Comment:
Just commenting on typo so @0xApotheosis doesn't have to 
```suggestion
  // the swapper has exceeded its rate limit
```

Reply:
english is hard :joy: